frontend/packages/component-library/src/carousel/carousel.module.scss

```diff
@@ -15,6 +15,7 @@
 .carouselWrapper {
   max-width: 100%;
   margin: 0 auto;
+  min-height: 50px;
```

Comment:
Added this here so there's workable space in the Contentful experiences editor, but it won't have an effect on anything visually once slides are added.

Reply:
super nit, but might be worth adding this comment in the code for the future